docs/source/projects/architecture/content-architecture.rst

```diff
@@ -102,6 +110,7 @@ to the notebook server, which saves it on disk as a JSON file with a
 ``.ipynb`` extension.
 
 .. image:: figs/notebook_components.png
+   :alt: Communication diagram showing five nodes labeled user, browser, notebook server, notebook file, and kernel. The notebook server is a communication hub. The browser, notebook file and kernel cannot talk to each other directly. They communicate indirectly through the notebook server. The user can only talk to the browser. The notebook server and kernel are colored green whereas the browser is colored purple to draw a distinction between the Jupyter front end versus back end.
```

Comment:
```suggestion
   :alt:
   
.. 
   Alt text is intentionally left blank because the image content is described thoroughly in the surrounding text.

The notebook server is a communication hub. The browser, notebook file on disk, and
kernel cannot talk to each other directly. They communicate through the notebook server. 
```